src/net.cpp

```diff
@@ -970,7 +970,6 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     CAddress addr;
     int nInbound = 0;
     int nMaxInbound = nMaxConnections - (nMaxOutbound + nMaxFeeler);
```

Comment:
Maybe it would be wise to cap this to 0 explicitly, as no one will be expecting this to be negative in future maintenance.
```
int nMaxInbound = std::max(nMaxConnections - (nMaxOutbound + nMaxFeeler), 0);
```